### ISSUE TITLE
How to replace the dedicated host? (stuck in strange state)

### ISSUE THREAD

**Issue by vkuusk** [Shippable/support#1109]
1. We had one dedicated host - everything worked fine. - OK
2. Disk was filled in 100% in /data/aufs - testruns started to fail - BAD 
3. I removed the dedicated host from shippable UI - but testruns were still started on the dedicated host and failed - BAD
4. I stopped the dedicated host (already unregistered from shippable) - testruns went into Shippable infrastructure - and finishing - OK
5. I rebuild a new server for dedicated host (used the same EIP) and initialized it in shippable. - OK
6. Now when I launch a new build test - It's not even passed to the dedicated host, times out and gets cancelled - BAD.

Question 1: Is this a bug or there is a procedure to remove a dedicated host from the shippable registry completely so I can replace a server on which it runs ? (How to delete from Shippable UI so it stops scheduling runs on it?)

Question 2: What is the procedure to replace a server which used as a the dedicated host ?

**Comment by manishas** [comment]
@vkuusk I just checked and I still see a host attached to your subscription...you will see it if you click on 'dedicated hosts' in the UI. You can delete the host by clicking on the trash can icon.

You can then add a new host. Let me know if you need help and I can get someone to help over a hangout.

**Comment by avinci** [comment]
@vkuusk there is a race condition of deletes. We are going to fix it on Monday.

For now, as a work around, ssh into the server you want to remove and run `docker rm -f $(docker ps -a)`. this should remove our agents and you can then delete it from the UI.

**Comment by avinci** [comment]
@vkuusk there is another issue of disk space. there is a bug in docker which is not releasing space. We are figuring out how to run maintenance on these hosts. One option is `sudo docker restart` which should release the space its holding. I personally don't like this solution and we realized this today that docker is having this issue. I will update more on what the solution is...

**Comment by vkuusk** [comment]
Sorry for the delay with testing on my side. I was out of town.

Today I added a new dedicated host today. It initialized OK, then it asked "Deploy Builder" this operation was stuck. and I used "Reset" option.

"Reset" completed and host went into state "Ready" "node is ready for builds"

But when I triggered the new build it hanged in "processing" state without producing anything else.

I left the host in this condition for you to check on your side before I'll try 
docker rm -f $(docker ps -aq)

Let whether or not you'll need to check the state of stuck build on shippable side: https://app.shippable.com/builds/550212fd8c20500c00bfcc20

Thanks
Regards,

Vlad
